app/Http/Controllers/ItemOrderController.php

```diff
@@ -68,7 +84,21 @@ public function edit($id)
      */
     public function update(Request $request, $id)
     {
-        //
+        $validationData = $request->validate([
+            'id' => 'required|int',
```

Comment:
Id is already a parameter look... (Request $request, $id). No need to validate id here.